tests/test_create_model.py

```diff
@@ -61,6 +61,17 @@ class Config(BaseModel.Config):
         model(foo=654)
 
 
+def test_custom_config_extras():
+    class Config(BaseModel.Config):
+        ignore_extra = False
+        allow_extra = False
+
+    model = create_model("FooModel", foo=(int, ...), __config__=Config)
```

Comment:
I haven't got around to finding a way to lint this, but pydantic should use single quotes always unless single quotes are used in the string.

Reply:
We struggled with this with black as well.  Ended up just using the Black convention.